vignettes/ospsuite.Rmd

```diff
@@ -11,96 +11,339 @@ vignette: >
-  : `IndividualCharacteristics`
-    : An object used for creating of individual parameter sets with the `createIndividual`-method. See [Creating individuals](create-individual.html) for more information.
-  : `Population`
-    : An object describing a virtual population. Can be either loaded from a *.csv-file created by PK-Sim or created with the `createPopulation`-method. See [Population simulations](create-run-population.html) for more information.
-  : `PopulationCharacteristics`
-    : An object used for creating of population parameter sets with the `createPopulation`-method. See [Population simulations](create-run-population.html) for more information.
-  : `SimulationPKAnalyses`
-    : PK analyses for a simulations result.
-  : `QuantityPKParameter`
-    : A certain PK parameter for an output quantity. Has the fields `$name` which is the name of the PK parameter (e.g. "C_max"), `$quantityPath` the path of the output the parameter has been calculated for, and `$values` the value (or list of values for a population simulation).
-  : `SensitivityAnalysis`
-    : A class defining the analysis of which input parameters have most impact on the output curves of a simulation. See [Sensitivity analysis](sensitivity-analysis.html) for more information.
-  : `SensitivityAnalysisResults`
-    : Results of running the `SensitivityAnalysis`
-  : `PKParameterSensitivity`
-    : The sensitivity (field `$value`) of a PK-Parameter (`$pkParameterName`) for the output `$outputPath` calculated for the varied parameter `$parameterName`. See [Sensitivity analysis](sensitivity-analysis.html) for more information.
+Call `print(object)` to display detailed information about any object.
+
+### Key Classes
+
+Understanding these main classes improves workflow effectiveness:
+
+**Simulation Objects:**
+
+- **`Simulation`**: Your loaded PBPK model from the .pkml file
+- **`SimulationResults`**: Output from running simulations
+- **`SimulationRunOptions`**: Controls for how simulations run (cores, error checking, progress bars)
+
+**Model Structure:**
+
+- **`Entity`**: Any part of the model (parameters, molecules, containers)
+- **`Container`**: Model compartments that contain other entities (organs, compartments)
+- **`Quantity`**: Entities with values (Parameters and Molecules)
+- **`Parameter`**: Model parameters you can modify
+- **`Molecule`**: Drug molecules and their concentrations
+
+**Results and Analysis:**
+
+- **`DataCombined`**: Container for combining simulation results and observed data
+- **`SimulationPKAnalyses`**: PK parameter calculations (AUC, Cmax, etc.)
+- **`Population`**: Virtual population data for population simulations
+
+**Configuration:**
+
+- **`OutputSchema`**: Defines when simulation results are saved
+- **`OutputSelections`**: Which quantities to include in results
+- **`SolverSettings`**: Numerical solver configuration
+
+## General Workflow Summary
```

Comment:
I wonder if some other important classes are worth mentioning in this overview, e.g. Sensitivity-related classes, `Application`, etc.

<img width="1040" height="327" alt="grafik" src="https://github.com/user-attachments/assets/a50d9f31-ba99-4b61-8a0a-974a42203cc5" />